docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -327,7 +327,7 @@ $ echo $dbs | grep -Eq '\bkeystone\b' && echo "OK" || echo "CHECK FAILED"
 $ echo $dbs | grep -Eq '\bneutron\b' && echo "OK" || echo "CHECK FAILED"
 $ echo "${PULL_OPENSTACK_CONFIGURATION_DATABASES[@]}" | grep -Eq '\bovs_neutron\b' && echo "OK" || echo "CHECK FAILED" <1>
 $ novadb_mapped_cells=$(oc exec openstack-galera-0 -n $NAMESPACE -c galera -- mysql -rs -uroot -p"${PODIFIED_DB_ROOT_PASSWORD['super']}" \
->   nova_api -e 'select uuid,name,transport_url,database_connection,disabled from cell_mappings;') <2>
+>   nova_api -e 'select uuid,name,transport_url,database_connection,disabled from cell_mappings where disabled=0;') <2>
```

Comment:
I think this is incorrect. We need to adopt disable computes as they can be enabled later. What the original bug report saw is that we are trying to adopt deleted computes. And I agree there, we should not try to adopt deleted computes.

Reply:
that makes sense